src/styles/components/display-menu.scss

```diff
@@ -0,0 +1,32 @@
+@import '../colours';
+@import '../settings';
+
+.display-menu {
+  & > ul > li > h5 {
```

Comment:
I have always had the impression that using Child Selector is not a great practice. This is particularly because it reduces the flexibility of the design and can be a lot more prone to bugs when the mark-up is changed. This is also why things like BEM and OOCSS are preferred way of writing CSS.

So, anyway we can get rid of this direct child selectors and write something no entirely bound to child descendant order?

Reply:
True - should be resolved